src/bitwise.rs

```diff
@@ -57,6 +60,15 @@ impl Bits for u32 {
     fn get_bit(self, bit_idx: u8) -> bool {
         self.is_bit_on(bit_idx)
     }
+
+    fn get_bits(self, bits_range: RangeInclusive<u8>) -> u32 {
+        let mut bits = 0b0;
+        for (shift_value, bit_index) in bits_range.enumerate() {
+            let bit_value: Self = self.get_bit(bit_index).into();
+            bits |= bit_value << shift_value;
+        }
+        bits
```

Comment:
We can write as, but not very important :)
```rust
        bits_range.into_iter().enumerate().fold(0, |bits, (v, i)| {
            let bit_value: Self = self.get_bit(i).into();
            bits | bit_value << v
        })
```

Reply:
Interesting. 

I thought that maybe get_bits would also work as a default implementation of the trait.  what do you think?

Reply:
Probably, 🤔